extensions/change-case/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "change-case",
+  "title": "Change Case",
+  "description": "Change the text in the clipboard",
```

I would be more specific here and propose something like this:
```suggestion
  "description": "Transform a string between camelCase, PascalCase, Capital Case, snake_case, param-case, CONSTANT_CASE and others",
```